bots/tests-scan

```diff
@@ -47,7 +47,7 @@ NPM_REPOS = {
 
 # Non-public images used for testing
 REDHAT_VERIFY = {
-    "verify/rhel-7": [ 'master', 'rhel-7.3.6', 'rhel-7.3.5', 'rhel-7.3.4', 'rhel-7.3.3', 'rhel-7.3.2' ],
+    "verify/rhel-7": [ 'master' ],
```

Comment:
That's actually wrong, we do want the rhel-7 test for testing the rhel-7.4 branch. But I realize that's unrelated to this PR, so I'll send another one after this.